docs/intro/faq.md

```diff
@@ -45,132 +45,132 @@ Telegram 群组链接为 [@OIwiki](https://t.me/OIwiki) ，QQ 群号码为 [ `58
 
-#### 我的数学公式怎么乱码了
+#### Why is my math formula garbled
 
-如果是行间公式（用的 `$$` ），目前已知的问题是需要在 `$$` 两侧留有空行，且 `$$` 要 **单独** 放在一行里（且不要在前加空格）。格式如下：
+If it is an inline formula (using `$$` ), the currently known problem is that you need to leave blank spaces on both sides of the `$$`, and `$$` should be put **individually** on one line (and DO NOT add spaces in the beginning). The format is as follows:
 
 ```text
-// 空行
+// Blank line
 $$
 a_i
 $$
-// 空行
+// Blank line
 ```
 
-#### 我的公式为什么在目录里没有正常显示？好像双倍了？
+#### Why is my formula not displayed properly in the catalog?
 
-是的，这个是 python-markdown 的一个 bug，可能近期会修复。
+Yes, this is a bug in python-markdown and would probably be fixed in the near future.
 
-如果现在想要避免目录中出现双倍公式，可以参考 <https://github.com/24OI/OI-wiki/blame/master/docs/string/sam.md#L82> 
+If you want to avoid double formulas in the catalog now, you can refer to <https://github.com/OI-wiki/OI-wiki/blame/master/docs/string/sam.md#L82>
 
 ```text
-### 结束位置 <script type="math/tex">endpos</script>
+### ending position <script type="math/tex">endpos</script>
 ```
 
-在目录中会变成
+In the directory will become
 
 ```text
-结束位置 endpos
+ending position endpos
 ```
 
-注：现在请尽量避免在目录中引入 MathJax 公式。
+Note: Please try to avoid introducing MathJax formulas in the catalog now.
 
-### 如何给一个页面单独声明版权信息
+### How to individually declare copyright information for a page
 
-参考 [Metadata](https://squidfunk.github.io/mkdocs-material/extensions/metadata/#usage) 的使用，在页面开头加一行即可。
+Please refer to the usage of [Metadata](https://squidfunk.github.io/mkdocs-material/extensions/metadata/#usage). Just add a line at the beginning of the page.
 
-比如：
+For example:
 
 ```text
 copyright: SATA
 ```
 
-注：默认的是‘CC BY-SA 4.0 和 SATA’。
+Note: The default is ‘CC BY-SA 4.0 and SATA’.
 
-### 如何给一个页面关闭字数统计（现已默认关闭）
+### How to turn off word count for a page (now closed by default)
 
-参考 [Metadata](https://squidfunk.github.io/mkdocs-material/extensions/metadata/#usage) 的使用，在页面开头加一行即可。
+Please refer to the usage of [Metadata](https://squidfunk.github.io/mkdocs-material/extensions/metadata/#usage). Just add a line at the beginning of the page.
 
-比如：
+For example:
 
 ```text
 pagetime:
 ```
 
-### 为什么作者信息统计处没有我的名字
+### Why does the author page does not have my name
 
-是因为 GitHub API 在文件目录变更后不能跟踪统计，我们在文件头手动维护了一个作者列表来解决这个问题。如果你发现自己写过一个页面中的部分内容，但是没有被记录在页面的作者去，可以把自己的 GitHub ID 加入到文件头的 `author` 字段，格式是 `author: Ir1d, cjsoft` ，相邻两个 ID 之间用 `,·` （逗号和空格）隔开。
+This is because the GitHub API cannot track statistics after the file directory changes. To solve this problem, We are manually maintaining a list of authors in the file header. If you have contributed to a part of a page, but not recorded by the author of the page, you can add your GitHub ID to the `author` field of the file header in the format of `author: Ir1d, cjsoft`. The two adjacent IDs are separated by `,·` (comma and space).
```

Comment:
```suggestion
This is because the GitHub API cannot track statistics after the file directory changes. To solve this problem, we are manually maintaining a list of authors in the file header. If you have contributed to a part of a page but not recorded, you can add your GitHub ID to the `author` field of the file header in the format of `author: Ir1d, cjsoft`. The two adjacent IDs are separated by `,·` (comma plus space).
```